include/swift/AST/Decl.h

```diff
@@ -1895,32 +1890,24 @@ class IfConfigDecl : public Decl {
   /// An array of clauses controlling each of the #if/#elseif/#else conditions.
   /// The array is ASTContext allocated.
   ArrayRef<IfConfigDeclClause> Clauses;
+  SourceLoc StartLoc;
```

Comment:
At one point I had caused destructive updates to the clause array, but that was ridiculous and I reverted it.  Going to revert this part too.